Week-6/6.3/assignment6.3.js

```diff
@@ -0,0 +1,22 @@
+function sort012(arr) {
+    let arr0 = [], arr1 = [], arr2 = [];
```

Comment:
@mr-rego Can you think about reusing the same array instead of creating new space like arr0, arr1 and arr2. Can you revisit the logic here ?